internal/pkg/source/entry.go

```diff
@@ -117,6 +118,37 @@ func (entries LazyLogEntries) Filter(term string) (LazyLogEntries, error) {
 	}, nil
 }
 
+// FilterRegEx filters entries by regex ignoring case
+func (entries LazyLogEntries) FilterRegExp(regex string) (LazyLogEntries, error) {
+	if regex == "" {
+		return entries, nil
+	}
+
+	re, err := regexp.Compile(strings.ToLower(regex))
```

Comment:
question: how will errors be reported? It would be annoying to loose everything after a small mistake in a regexp